### PR TITLE
Update the -vs switch and add docs for it

### PR DESCRIPTION
**Pull request by ViktorHofer**
We don't need to point to the testhost folder anymore (for libraries) to support VS Test Explorer.

### CODE REVIEW

**Comment by AntonLapounov** [comment]
Could you please also fix this line? The short form `-v` does not work, because PowerShell binds it to the `$vs` parameter instead of `$properties`. https://github.com/dotnet/runtime/blob/08e04457baeb8cb1aa96687425d1ff4855d925aa/eng/build.ps1#L29

File: eng/build.ps1 [hunk 1]
```diff
@@ -59,38 +59,32 @@ if ($help -or (($null -ne $properties) -and ($properties.Contains('/help') -or $
...
-  Write-Host "VS Test Explorer now works without needing to call build.cmd. The -vs switch will be removed eventually."
   . $PSScriptRoot\common\tools.ps1
 
-  # Microsoft.DotNet.CoreSetup.sln is special - hosting tests are currently meant to run on the
-  # bootstrapped .NET Core, not on the live-built runtime.
-  if (([System.IO.Path]::GetFileName($vs) -ieq "Microsoft.DotNet.CoreSetup.sln") -or ($vs -ieq "Microsoft.DotNet.CoreSetup")) {
+  if (-Not (Test-Path $vs)) {
+    $solution = $vs
+    # Search for the solution in libraries
+    $vs = Join-Path (Join-Path (Split-Path $PSScriptRoot -Parent) "\src\libraries") $vs | Join-Path -ChildPath "$vs.sln"
```
Comment (AntonLapounov) [inline eng/build.ps1:68]: Nit: I would stick to either parentheses or pipes, but I would not mix those two styles.  Either:
```ps1
Join-Path (Join-Path (Join-Path (Split-Path $x -Parent) "src\libraries") $vs) "$vs.sln"
```
or
```ps1
Split-Path $x -Parent | Join-Path -ChildPath "src\libraries" | Join-Path -ChildPath $vs | Join-Path -ChildPath "$vs.sln"
```
(The backslash before `src` may be omitted.)

File: docs/workflow/requirements/windows-requirements.md [hunk 2]
```diff
@@ -71,6 +71,14 @@ While not strictly needed to build or test this repository, having the .NET SDK
 We use this in the [Using Your Build](../testing/using-your-build.md) instructions.
 The minimum required version of the SDK is specified in the [global.json file](https://github.com/dotnet/runtime/blob/master/global.json#L3). [You can find the installers and binaries for nightly builds of .NET SDK here](https://github.com/dotnet/installer#installers-and-binaries).
 
+Alternatively to avoid modifying your machine state, you can use the repository's locally acquired SDK by passing in the solution to load via the `-vs` switch:
```
Comment (AntonLapounov) [inline docs/workflow/requirements/windows-requirements.md:74]: Nit: We may need a comma after "Alternatively".

**Review by AntonLapounov — APPROVED** [review]

**Comment by ViktorHofer** [comment]
> Could you please also fix this line? The short form -v does not work, because PowerShell binds it to the $vs parameter instead of $properties.

Thanks, I decided to instead fix the -v alias.

**Comment by ViktorHofer** [comment]
Unrelated failure:
```
Failed to download package 'System.Collections.Immutable.1.7.1' from 'https(0,0): error : (NETCORE_ENGINEERING_TELEMETRY=Restore) 1408F119:SSL routines:SSL3_GET_RECORD:decryption failed or bad record mac
```

**Pull request merged**